src/components/data-table/data-table-story.mdx

```diff
@@ -73,7 +73,7 @@ import 'carbon-web-components/es/components/data-table/index.js';
 
 Adding `selection-name` attribute to `<bx-table-row>`/`<bx-table-header-row>` shows the selection UI in the table row.
 
-When user attempts to change row selection, the following events fire:
+The logic of row selection needs to be handled on by the application code. When user attempts to change row selection, the following events fire:
```

Comment:
Wondering if we want to reiterate here that they can see an example of this in our codesandbox?

Reply:
I'll open up a ticket to capture the row selection codesandbox example